docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,341 @@
+
+# MLflow 3.0
+
+Discover the next generation of MLflow, designed to streamline your AI experimentation and accelerate your journey from idea to production. MLflow 3.0 brings cutting-edge support for GenAI workflows, enabling seamless integration of generative AI models into your projects.
+
+## What is MLflow 3.0?
+
+The best AI experimentation platform for building high quality models and GenAI agents! MLflow 3.0 introduces a variety of exciting improvements to MLflow Tracking and Tracing, including:
+- First class model support in MLflow tracking with **LoggedModel**
+- **Model-centric** information architecture for params and metrics in MLflow tracking
+- Enhanced **tracing** experience with lineage tracking for LoggedModels
+
+Model Tracking helps you create and evaluate different model/agent configurations in your experiments, LoggedModel helps you find the best ones for production usage, and tracing helps enhance observability in your model or GenAI applications.
+
+
+## Model Tracking with LoggedModel
+
+In MLflow 3.0, we introduce a new model-centric design, together with revamped APIs (UI will come later), tailored to enhance deep learning and generative AI workflows. 
+In deep learning, training often generates multiple model checkpoints, where the best candidates are further evaluated before production deployment. 
+With GenAI agents, there are multiple rounds of offline evaluation via batch jobs and interactive queries from human beta testers. 
+
+We are introducing a new first-class object, the Logged Model entity, into MLflow Tracking to streamline these processes. As your deep learning jobs create and evaluate models, 
+or you define and evaluate your GenAI agents in code, they will be automatically stored as MLflow Logged Models in your MLflow Experiment. 
+
+Every deep learning checkpoint is stored as a Logged Model with direct links to its own metrics. There is no longer any need to carefully log evaluation metrics for each checkpoint
+at its epoch all within the same Run, or to later query all metrics at a certain epoch just to retrieve metrics for one checkpoint. 
+Similarly, each GenAI agent consolidates its evaluation metrics from offline jobs, and online interactions if you use managed MLflow with Databricks. 
+This feature removes the need to query Runs for traces from different environments (batch jobs, separate inference tables etc.) to gain a full understanding of the agent’s performance.
+
+Specifically, you will be able to:
+- View Logged Models alongside key information, including code and weights as artifacts and configurable parameters that affect model behavior
+- Holistically track all performance metrics related to your logged models with links to training and evaluation jobs
+- Search for and rank existing models based on a set of certain criteria, such as performance metrics on training or evaluation datasets
+
+Simple example:
+
+```python
+import mlflow
+
+
+class MyModel(mlflow.pyfunc.PythonModel):
```

Comment:
Since all of our recommended paths for model logging for GenAI are going to be focused on models from code, should we update this example to highlight that? It would good to reinforce this paradigm in all examples, even though it is slightly more complex.

Reply:
Since mlflow-3 is not only for GenAI, and this example is super simple maybe we can keep it as it is? I'll update the GenAI one to use models from code though!

Reply:
@BenWilson2 @serena-ruan instead of the "hello world" example, I'd prefer if we show a compelling *screenshot* of a GenAI example containing LoggedModel params and metrics. I don't think there's a ton of value in showing "hello world" examples since we have full GenAI / DL examples on the next page, since they aren't indicative of real workflows and can make the product appear "out of touch"